packages/user-management-widget/src/lib/widget/state/selectors.ts

```diff
@@ -12,9 +13,18 @@ export const getUsersList = createSelector(getRawUsersList, (users) =>
   users.map((user) => ({
     ...user,
     status: userStatusMappings[user.status] || user.status,
+    roles: user.roleNames
+      .concat(user.userTenants?.[0]?.roleNames)
+      .filter(Boolean)
+      .sort()
+      .join(', '),
```

Comment:
I don't think we should join it here, or join it at all
AFAIR, we can pass an array to the grid and it should be ok